### DISCUSSION TITLE
Allow tags for repositories to organize repos within an organization

### DISCUSSION THREAD

**Discussion by DonDebonair** [community/community#5387] [Other Feature Feedback, Questions, & Ideas]
Github sadly doesn't supported nested organizations/folders [like Gitlab does](https://docs.gitlab.com/ee/user/group/subgroups/), so when you have an organization with many repos, it becomes unwieldy to find them. It would be nice if you could tag repos and filter repos within an organization on those tags. Kinda similar to _topics_, but for repos within an org.

**Comment by MarcoValliIMA** [comment]
There's also a similar discussion here: #5387 and here #4147
